client/src/components/profile-page/Profile.css

```diff
@@ -4,111 +4,139 @@
 }
 
-.profile-wrapper {
+.grid-wrapper {
+  margin-top: 2em;
+  max-height: 100vh;
+  max-width: 100vw;
+  display: grid;
+  grid-template-rows: 25vh 25vh 25vh;
+  grid-template-columns: 50vw 50vw;
+}
+.top-profile-wrapper {
+  grid-area: 1/1/2/3;
+  display: flex;
   position: relative;
-  height: 80vh;
 }
 
-.profile-picture {
-  height: 200px;
-  position: absolute;
-  left: 5px;
-  top: 39.5px;
-  border-radius: 50%;
+.top-text-wrapper {
+  display: flex;
 }
 
-.profile-bubble {
-  position: absolute;
-  left: 150px;
-  top: 25.67px;
-  height: 200px;
+.profile-picture {
+  height: 100%;
+  border-radius: 50%;
 }
-
-.profile-bio {
-  position: absolute;
-  top: 220.9px;
-  left: -15px;
-  height: 170px;
-  z-index: 1;
+.top-wrapper-icons {
+  display: flex;
+  flex-direction: column;
+  height: 60%;
 }
-
-.profile-bio-text {
-  font-family: Arial, Helvetica, sans-serif;
-  font-size: 10;
-  color: var(--deep-blue);
-  width: 200px;
-  height: 26px;
-  position: absolute;
-  left: 39.79px;
-  top: 265px;
-  z-index: 2;
+.top-wrapper-text {
+  display: flex;
+  flex-direction: column;
+  z-index: 10;
+  width: 10rem;
+  height: 60%;
 }
 
 .fren-icon {
-  position: absolute;
-  width: 39.13px;
-  height: 36.69px;
-  left: 189.44px;
-  top: 40px;
-  z-index: 1;
+  z-index: 10;
+  font-size: 2.5rem;
 }
-
 .bone-icon {
-  position: absolute;
-  width: 37.68px;
-  height: 25.84px;
-  left: 189.44px;
-  top: 90px;
+  font-size: 2.5rem;
+  z-index: 10;
   transform: rotate(-45deg);
-  z-index: 1;
-  color: var(--light-yellow);
+  /* color: var(--light-yellow); */
 }
 
 .fren-count {
   font-family: "Cookie", cursive;
-  position: absolute;
-  left: 237.27px;
-  top: 40px;
-  z-index: 1;
-  font-size: 36px;
+  font-size: 2.5rem;
   color: var(--deep-blue);
 }
+
 .dog-rating {
   font-family: "Cookie", cursive;
-  position: absolute;
-  left: 237.27px;
-  top: 90px;
-  z-index: 1;
-  font-size: 36px;
+  font-size: 2.5rem;
   color: var(--deep-blue);
 }
 
+.profile-bubble {
+  height: 130%;
+  margin-left: -14rem;
+  margin-top: -1rem;
+}
+
+.middle-profile-wrapper {
+  grid-area: 2/1/3/3;
+  display: flex;
+  position: relative;
+}
+
+.middle-text-wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 35em;
+  height: 90%;
+  margin-left: -15.5em;
+  z-index: 10;
+  align-self: center;
+}
+
 .dog-name {
   font-family: "Cookie", cursive;
-  position: absolute;
-  left: 183.06px;
-  top: 225.77px;
-  z-index: 2;
   font-size: 37px;
   color: var(--deep-blue);
   font-weight: 500;
+  text-align: right;
+}
+
+.profile-bio-text {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 75%;
+  color: var(--deep-blue);
+}
+.profile-bio {
+  height: 110%;
+  margin-left: -1em;
 }
 
-.tour-wrapper {
+.feedback-list-wrapper {
   display: flex;
-  justify-content: center;
-  position: relative;
+  flex-direction: column;
+  width: 100%;
+  text-align: center;
+  align-self: flex-end;
+  margin-bottom: -2em;
+  z-index: 10;
+}
+
+.feedback {
```

Comment:
I would add a `margin` here, so the headline isn't so close to the bubble _(only stylistic advice haha)_